### DISCUSSION TITLE
Reading npz or npy files with tract

### DISCUSSION THREAD

**Discussion by igor-yusupov** [sonos/tract#1215] [Q&A] [answered]
I have saved numpy array and I want to use it, can I read npz or npy files with tract?

**Comment by kali** [comment, marked as answer]
You an use tract-libcli tensors::for_npz. It's just a dump wrapper around ndarray_npy if you prefer not to pull tract-libcli.

https://github.com/sonos/tract/blob/main/libcli/src/tensor.rs#L200

**Comment by igor-yusupov** [comment]
@kali it seems to be the same as using ndarray_npy::NpzReader. I got it, thank you!
